components/dashboard/src/user-settings/PersonalAccessTokens.tsx

```diff
@@ -169,8 +169,8 @@ function ListAccessTokensView() {
         <>
             <div className="flex items-center sm:justify-between mb-4">
                 <div>
-                    <Heading2>
-                        Access Tokens{" "}
+                    <Heading2 className="flex gap-4 items-center">
+                        Access Tokens
```

Comment:
This is a minor style change for the PAT Token page to have a consistent design :)